taichi/analysis/value_diff.cpp

```diff
@@ -102,11 +95,6 @@ class ValueDiff : public IRVisitor {
 
 namespace irpass::analysis {
 
-DiffRange value_diff(Stmt *stmt, int lane, Stmt *alloca) {
```

Comment:
So `value_diff` is never used now? I wonder if we should use `value_diff` to check if two global pointers are definitely different (e.g. `a[i, j]` and `a[i + 1, j]` are different if analyzed with `value_diff`, but we can conclude nothing if analyzed with `same_statements`.)

Reply:
Right, it's never used now, except for on loop indices... We should recover it in the future for stronger aliasing analysis :-)